packages/react/src/hooks/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './useMediaQuery';
+export * from './useMobile';
+export * from './useTheme';
```

Comment:
> I guess this will export other hooks as well?
Exports only what is mentioned here in the file. All the other hooks look like they're imported but they aren't actually usable. It's weird.
> I'm not sure how useTheme should work. 
Ah maybe I should remove that altogether from here then. Better keep it just for us, for now 😄

Reply:
I removed both of these because users don't really have use for them (as far as we know). I don't remember why I thought that they would have, I probably confused these with something else.